apimodels/agent_models.go

```diff
@@ -243,6 +246,9 @@ func (ch *CreateHost) validateEC2() error {
 			catcher.New("subnet ID must be set if AMI is set")
 		}
 	}
+	if ch.Tenancy != "" {
+		catcher.NewWhen(ch.Tenancy != "default" && ch.Tenancy != "dedicated" && ch.Tenancy != "host", "if set, tenancy must be 'default', 'shared', or dedicated'")
```

Comment:
nit: would it be worth making consts for these 3 things? maybe not haha

Reply:
Hmm yeah it's a good idea. I added constants.